docs/guides/key-manager/upgrade-lsp6.md

```diff
@@ -0,0 +1,246 @@
+---
+sidebar_label: 'Upgrade LSP6 Key Manager'
+sidebar_position: 4
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Upgrade the LSP6 Key Manager
+
+:::info Requirements
+
+You will need a Universal Profile that you can control via its KeyManager to follow this guide.
+If you don't have a Universal Profile yet, follow our previous guide [**Create a Universal Profile**](../universal-profile/create-profile.md) or look at the [_lsp-factory.js_](../../tools/lsp-factoryjs/deployment/universal-profile.md) docs.
+
+:::
+
+In this guide, we will learn how to upgrade the LSP6 Key Manager of your Universal Profile to the latest version available.
+
+By the end of this guide, you will know how to:
+
+- Deploy a new LSP6 Key Manager with the last updates.
+- Transfer ownership of your Universal Profile through the old LSP6 Key Manager.
+- Accept ownership of your Universal Profile through the new LSP6 Key Manager
+
+## Setup
+
+```shell
+npm install @lukso/lsp-smart-contracts web3
+```
+
+## Step 1 - Set up constants
```

Comment:
Maybe we can integrate Step 1 and 2 together?

It might be easier to just put three constant values in the same single file.

So the code snippet below would look like this:

```js
import LSP6KeyManager from '@lukso/lsp-smart-contracts/artifacts/LSP6KeyManager.json' assert { type: 'json' };
import Web3 from 'web3';
const web3 = new Web3('https://rpc.l14.lukso.network');

const privateKey = '0x...';
const universalProfileAddress = '0x...';
const keyManagerAddress = '0x...';
```